frontend/src/metabase/components/DebouncedFrame.jsx

```diff
@@ -32,12 +32,18 @@ export default class DebouncedFrame extends React.Component {
     };
   }
 
-  setSize = (width, height) => {
+  updateSize = () => {
```

Comment:
Rename the function to indicate that this isn't really a setter, but a function to update/reset the size so it's not accepting any arguments. I'm not sure if `resetSize` would be a better name, but `resetSize` could also sound like the state will be reset to `null`